### ISSUE TITLE
Mitotic Classifier

### ISSUE THREAD

**Issue by tknijnen** [AllenCellModeling/long-term-eng#3]
## Use Case
The mitotic classifier is of importance to the institute for various projects. There is renewed interest because of the updated segmentation abilities. The current version of the mitotic classifier is not a modular, scalable, versionable, shareable workflow. The mitotic classifier is an excellent use case to drive the development of modular, scalable, versionable, shareable infrastructure for cross-team science.   

## Solution
Use a data+process visual diagram and the [cookiecutter-stepworkflow](https://github.com/AllenCellModeling/cookiecutter-stepworkflow) to rewrite the mitotic classifier.
![image](https://user-images.githubusercontent.com/4379993/76158094-e6c9d700-60c6-11ea-8df7-7d2b876333cb.png)
![image](https://user-images.githubusercontent.com/4379993/76158101-f9dca700-60c6-11ea-8fd1-33cafe9cb5cd.png)

## Alternatives
Keep the classifier as is. Be able to run the old classifier code on the new cells.

## Stakeholders
Theo, Jianxu, Jamie, Rory, Jackson

## Major Components
[ ] Make a data+process visual diagram of the complete mitotic classifier
[ ] Rewrite using stepworkflow
[ ] Enable distributed GPU compute for training on DGX boxes

## Dependencies
_Please add any other major or minor project dependencies here_

## Other Notes
This project still needs to be discussed with AD, and get the green light (and more stakeholders) before it can commence.

**Comment by evamaxfield** [comment]
We should pull in Julie T. I mentioned the project to her and she had some ideas on how it should be rewritten. In short she emphasized that if possible it would be great to use mitotic aligned timeseries data rather than still frame images for classification.

**Comment by heeler** [comment]
I totally agree. Also I think it should be more in line w/ a continuous-time classification and leverage J. Ellenburg's publications.

**Comment by donovanr** [comment]
I would be happy to supervise the ML side of this, but I don't think it makes sense to make a push on this until the inputs (both promised and extant) are more clear.

The process would be quite different for one shot vs online learning, singe data source vs mixed pipeline + time-series, etc.

**Comment by evamaxfield** [comment]
Updates on this: See [mito_class](https://github.com/allencellmodeling/mito_class) for early implementations and experiments.
